exercise/staff_views.py

```diff
@@ -179,7 +183,11 @@ class AllResultsView(CourseInstanceBaseView):
     def get_common_objects(self):
         super().get_common_objects()
         self.table = ResultTable(self.instance)
```

Comment:
This can be dropped as it's not used. Am I right?

Reply:
`ResultTable` builds the whole result table in the server side, i.e., fetches all submission data in the course from the database. Since the new results page fetches the data from the REST API, ResultTable is absolutely unnecessary in the view and must be removed.